### PR TITLE
Feat/ens registrar

### PR DESCRIPTION
**Pull request by tspoff**

### CODE REVIEW

**Comment by pimato** [comment]
can you guys make that modal scrollable for the categories:
Bildschirmfoto 2020-03-08 um 19 44 13

**Comment by tibetsprague** [comment]
I wonder if just making the text and padding smaller in this case would be enough. also could make the tab text a bit more concise. like "Transfer ownership of token ID" seems clear enough for example

**Comment by dOrgJelli** [comment]
@tibetsprague @pimato fixed:

**Zoomed Out:**
image

**Zoomed In:**
image
image

**CSS Changes:**
image

**Comment by pimato** [comment]
amazing, thanks @dOrgJelli !

**Comment by tibetsprague** [comment]
great! I still think the tab text can be shortened up :) only other request id make is for the background of the sidebar to be all one color, either white or the darker gray you changed it to. This looks better and makes it clearer which tab is selected

**Comment by dOrgJelli** [comment]
@tibetsprague let's do this in another PR, as this one is simply to add support for the ENS Registrar and does not make any styling changes (please see files changed).

**Comment by tibetsprague** [comment]
hmm, why is the background gray for you? weird. ok...

**Comment by dOrgJelli** [comment]
can we merge this @tibetsprague?

**Comment by tibetsprague** [comment]
done!

File: src/genericSchemeRegistry/schemes/EnsRegistrar.json [hunk 1]
```diff
@@ -0,0 +1,560 @@
+{
+  "name": "EthRegistrar",
```
Comment (dkent600) [inline src/genericSchemeRegistry/schemes/EnsRegistrar.json]: EthRegistrar?  Shouldn't be EnsRegistrar?

**Pull request merged**